packages/account/package.json

```diff
@@ -32,6 +32,7 @@
     "@deriv/components": "^1.0.0",
     "@deriv/shared": "^1.0.0",
     "@deriv/translations": "^1.0.0",
+    "bowser": "^2.9.0",
```

Comment:
Can we fulfil what we need from `bowser` using our local `browser_detect.js`? (Or is it reasonable to make the changes to our util file to get what we need?)

`bowser` when minified and gzipped is `5.6KB`, while our own util is ~400 bytes.

Reply:
Per our discussion, we'll continue using `bowser`. Also we're going to keep our `browser_detect.js` for now until deciding what to do with `performance-checker`.